packages/core/src/actions/getBalance.ts

```diff
@@ -18,12 +18,11 @@ export async function getBalance<
   client: Client<Transport, C, A, UTXOSchema>,
   params: GetBalanceParameters
 ): Promise<GetBalanceReturnType> {
-  const data = await client.request(
+  return client.request(
```

Comment:
Why did we change that? It was easier to debug 😁

Reply:
putting it back